### PR TITLE
vendor updates

### PR DESCRIPTION
**Pull request by Guite**
| Q                 | A
| ----------------- | ---
| Bug fix?          | no
| New feature?      | no
| BC breaks?        | no
| Deprecations?     | no
| Fixed tickets     | #2661
| Refs tickets      | -
| License           | MIT
| Changelog updated | yes

### CODE REVIEW

File: CHANGELOG-1.4.md [hunk 1]
```diff
@@ -50,6 +51,7 @@ CHANGELOG - ZIKULA 1.4.x
     - symfony updated to from 2.8.4 to 2.8.8
     - swiftmailer updated from 5.4.1 to 5.4.3
     - components/jquery-ui updated from 1.11.4 to 1.12.0
+    - beberlei/doctrine-extensions updated from 0.3 to 1.0.11
```
Comment (craigh) [inline CHANGELOG-1.4.md]: that's a pretty significant upgrade. is BC maintained?

Comment (cmfcmf) [inline CHANGELOG-1.4.md]: > If you're still using Paginator, LargeCollections, Phing, PHPUnit or Versionable behaviours available in 0.1–0.3, you're welcome to use 0.3 – but do note, this functionality is now available in Doctrine core, no longer supported in this library, and was removed in 1.0.

:-1: on the update. The current version is only there for BC and subject to removal in 2.0.

Comment (craigh) [inline CHANGELOG-1.4.md]: > 👎 on the update. The current version is only there for BC and subject to removal in 2.0.

I agree with @cmfcmf . @Guite please revert this change

File: CHANGELOG-1.4.md [hunk 2]
```diff
@@ -42,6 +42,7 @@ CHANGELOG - ZIKULA 1.4.x
  - Vendor updates:
     - font awesome updated from 4.5.0 to 4.6.3
     - jms translation bundle updated from 1.2.1 to 1.2.3
+    - js routing bundle updated from 1.5.x to 1.6.0
```
Comment (craigh) [inline CHANGELOG-1.4.md]: any BC breaks in this upgrade?

Comment (Guite) [inline CHANGELOG-1.4.md]: nope

File: CHANGELOG-1.4.md (hunk 1 above)
Comment (Guite) [inline CHANGELOG-1.4.md]: done

**Comment by craigh** [comment]
so - changing back to jms vendor. did you look into any differences or do any testing?

**Comment by Guite** [comment]
I've looked into differences and found nothing noteworthy.
